server/bleep/src/webserver.rs

```diff
@@ -128,19 +127,17 @@ where
     Json(Response::from(val))
 }
 
-pub(in crate::webserver) fn error<'a>(
-    kind: ErrorKind,
-    message: impl Into<Cow<'a, str>>,
-) -> Json<Response<'a>> {
+pub(in crate::webserver) type Result<T, E = Error> = std::result::Result<T, E>;
```

Comment:
Did you know that `pub(in ..)` might be deprecated in the 2024 edition?

Reply:
I didn't know that 😄 It's not particularly useful here, I'm just following the visibility specifier that other types use.